Cargo.toml

```diff
@@ -85,7 +85,7 @@ tokio = { version = "1.28", features = [] }
 
 # pearl
 [workspace.dependencies.pearl]
-version = "0.19.0"
+version = "0.20.0"
```

Comment:
This is temporary change. https://github.com/qoollo/bob/pull/848 should be merge first